RecoTracker/TkNavigation/BuildFile.xml

```diff
@@ -0,0 +1,5 @@
+<use name="RecoTracker/TkDetLayers"/>
+<use name="TrackingTools/DetLayers"/>
+<export>
+  <lib   name="1"/>
+</export>
```

Comment:
Apparently this file was not needed before.

what does this do for a header-only package?

@makortel @Dr15Jones

Reply:
> Apparently this file was not needed before.
> 
> what does this do for a header-only package?
> 
> @makortel @Dr15Jones

@smuzaffar Could you clarify?

Reply:
BuildFiles are useful for header only packages too. This way one exports the dependency so that the packages which use this do not have to explicitly add the missing dependencies. The `<export> </export>` section in this type of BuildFile(s) does not make any sence as there is no library generated for such packages. SCRAM silently ignores this. Better to remove the  `<export> </export>`  section here.

Reply:
Thanks, that's good too know! I was checking the wrong header-only package for inspiration: https://github.com/cms-sw/cmssw/tree/master/RecoCaloTools/Selector

@slava77, as this is in the reco area, how do you want to proceed? We can either change this now, or we rely for now on scram ignoring the export statement and remove it in another PR (which can also cover the other header-only libraries). I favour the latter solution, because then it doesn't reset all the signatures and we are almost there...